Modules/install-jaspModuleInstaller.R.in

```diff
@@ -94,7 +94,9 @@ renv::restore(lockfile = lf, library = .libPaths(), rebuild = pkgName)
 
 
 cat("jaspModuleInstaller::writeModuleStatusObject\n")
-jaspModuleInstaller::writeModuleStatusObject("@PROJECT_SOURCE_DIR@", oldModuleStatusObject, "@MODULES_BINARY_PATH@")
+modulesToInstallRaw <- "@JASP_COMMON_MODULES@;@JASP_EXTRA_MODULES@"
+modulesToInstall <- strsplit(modulesToInstallRaw, ";", fixed = TRUE) # should perhaps use CMAKE_PREFIX_PATH
```

Comment:
So are you removing all the `install-module.R.in` ?

Reply:
The whole `foreach(MODULE ${JASP_COMMON_MODULES})` thing

Reply:
(I guess it would be better to generate the hashes in those `install-module.R.in` gen'ed scripts instead if all at once at the beginning.

Reply:
no, this is (for now) just to generate the module status object. This information here makes it a bit easier to print in advance which modules need updates. Previously, it just looked at all possible modules in jasp-root/Modules. But that is misleading, because then if you enabled jasp_test_modules it would always print that e.g., jaspCochrane needs to be installed.

Reply:
> (I guess it would be better to generate the hashes in those install-module.R.in gen'ed scripts instead if all at once at the beginning.

Not really. Because then every module that depends on jaspDescriptives is going to re-hash jaspDescriptives, which is exactly what I want to avoid.